tests/test_matching.py

```diff
@@ -99,11 +103,99 @@ def test_iterative_match_categorical(setup_data):
     assert result == expected_result
 
 
-@pytest.mark.skip(reason="todo")
-def test_match_individuals():
-    pass
+def get_test_dfs_and_mapping(mapping):
+    samples = [
+        [0, 1, 3, 1, 0],
+        [1, 2, 0, 2, 1],
+        [2, 2, 1, 1, 2],
+        [3, 2, 2, 2, 3],
+        [4, 3, 0, 1, 4],
+        [5, 3, 1, 2, 5],
+        [6, 3, 3, 1, 6],
+        [7, 4, 1, 2, 7],
+        [8, 4, 1, 1, 8],
+        [9, 4, 3, 2, 9],
+    ]
+    columns = ["id", "hid", "age_group", "sex", "eco_stat_0"]
+    return (
+        pd.DataFrame(
+            samples,
+            columns=columns,
+        ),
+        pd.DataFrame(
+            [
+                # Use eneumerate for ids, keep hids in same order
+                [id, samples[id][1], *samples[idx][2:]]
+                for id, idx in enumerate(mapping)
+            ],
+            columns=columns,
+        ),
+        mapping,
+    )
 
 
-@pytest.mark.skip(reason="todo")
-def test_match_psm():
-    pass
+@pytest.fixture
+def ind_hh():
```

Comment:
Might be helpful to add a comment explaining these fixtures

Reply:
Added in [68955b8](https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/91/commits/68955b87d0c74751dded2d3e6c926ec704c7cc48)